osu.Game/Database/StandardisedScoreMigrationTools.cs

```diff
@@ -264,9 +266,63 @@ public static long ConvertFromLegacyTotalScore(ScoreInfo score, LegacyBeatmapCon
             switch (score.Ruleset.OnlineID)
             {
                 case 0:
+                    if (score.MaxCombo == 0 || score.Accuracy == 0)
+                    {
+                        return (long)Math.Round((
+                            0
+                            + 300000 * Math.Pow(score.Accuracy, 8)
+                            + bonusProportion) * modMultiplier);
```

Comment:
This appears to not be matching the latest comment? I'd expect to see

```suggestion
                        return (long)Math.Round((
                            0
                            + 500000 * Math.Pow(score.Accuracy, 5)
                            + bonusProportion) * modMultiplier);
```
That said I'm not even going to pretend I can tell because this PR thread is a shitshow and I don't know what's even up to date anymore.

Reply:
Yes you're correct, I forgot to modify this. Mybad.

Reply:
Dunno. If anything, private method that takes the portions and spits out the result. But I'm also fine with just fixing locally.